torchtune/training/checkpointing/_checkpointer.py

```diff
@@ -136,41 +146,63 @@ def __init__(
         recipe_checkpoint: Optional[str] = None,
         resume_from_checkpoint: bool = False,
     ) -> None:
+
         # Fail fast if ``checkpoint_files`` is invalid
+        # FIXME TODO
```

Comment:
?

Reply:
changed to "# TODO: support loading more than one file"